src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.ServiceModel.Tests.Common;
 
 public static class BaseAddress
 {
```

Comment:
Nit: BaseAddress is currently used by unit tests too, and we don't want the Bridge to be required to run unit tests. No actual client to server communication occurs over the raw BaseAddress URLs.  Endpoints.cs was meant to be used by scenario tests that needed a running server.  It effectively layered over BaseAddress.  I think we could just revert BaseAddress to its old form now that Endpoints does not depend on it anymore (and let the unit tests run without the Bridge.